src/app/app.module.ts

```diff
@@ -131,7 +143,19 @@ export const CUSTOM_FORMATS = {
     MembershipsNewComponent,
     MembershipsEditComponent,
     PermissionsComponent,
-    AuthorizedUsersSelectComponent
+    AuthorizedUsersSelectComponent,
+    WelcomeMessageCardComponent,
+    MessageCardComponent,
+    DeletableNotificationComponent,
```

Comment:
If we remove `@Component` directive from the class `DeletableNotificationComponent` there will be no need to register it within `AppModule` and we can remove this line of code.